libminifi/src/core/yaml/YamlConfiguration.cpp

```diff
@@ -32,16 +32,25 @@ namespace core {
 core::ProcessGroup *YamlConfiguration::parseRootProcessGroupYaml(
     YAML::Node rootFlowNode) {
   uuid_t uuid;
+  int64_t version = 0;
 
   checkRequiredField(&rootFlowNode, "name", CONFIG_YAML_REMOTE_PROCESS_GROUP_KEY);
   std::string flowName = rootFlowNode["name"].as<std::string>();
   std::string id = getOrGenerateId(&rootFlowNode);
   uuid_parse(id.c_str(), uuid);
 
-  logger_->log_debug("parseRootProcessGroup: id => [%s], name => [%s]", id,
-                     flowName);
+  if (rootFlowNode["version"]) {
+    std::string value = rootFlowNode["version"].as<std::string>();
+    if (core::Property::StringToInt(value, version)) {
+      logger_->log_debug("parseRootProcessorGroup: version => [%d]", version);
+    }
+  }
+
+  logger_->log_debug(
```

Comment:
thanks for using the string. i saw a few places where it was id.c_str() if you could make them consistent that would be awesome.